src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -317,6 +367,18 @@ const WorkflowView = _.flow(
     ])
   }
 
+  async fetchDescription() {
+    const { methodRepoMethod: { methodNamespace, methodName, methodVersion } } = this.state.savedConfig
+    const { ajax: { Methods } } = this.props
+    const synopsis = await (() => {
+      return Methods.method(methodNamespace, methodName, methodVersion).get().then(({ synopsis }) => synopsis)
+    })()
+    const documentation = await (() => {
+      return Methods.method(methodNamespace, methodName, methodVersion).get().then(({ documentation }) => documentation)
+    })()
```

Comment:
I see that we're already hitting this endpoint when we fetch the WDL, so it looks like the right thing to do would be to fetch all of this data preemptively on load (and then not do the whole lazy WDL thing). Also, what you have only works for Agora methods--is this info available from Dockstore too?

Reply:
It's not available from Dockstore to my knowledge